### PR TITLE
dVRK Control via Socket

### PR DESCRIPTION
**Pull request by AkhilDeo**
Create an example for controlling AMBF surgical_robotics_challenge PSMs wirelessly via sockets. Includes detailed README for usage.

### CODE REVIEW

File: ambf_ros_modules/examples/dvrk_control_socket_example/README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# dVRK Control via Socket
```
Comment (AkhilDeo) [inline ambf_ros_modules/examples/dvrk_control_socket_example/README.md:1]: Not technically the dVRK, should I be referring to simulation as dVRK?

Comment (adnanmunawar) [reply ambf_ros_modules/examples/dvrk_control_socket_example/README.md:1]: Hello Akhil, thanks for this PR. I think this one is more relevant to the SRC. What do you think?

Comment (AkhilDeo) [reply ambf_ros_modules/examples/dvrk_control_socket_example/README.md:1]: What do you mean by SRC?

Comment (adnanmunawar) [reply ambf_ros_modules/examples/dvrk_control_socket_example/README.md:1]: It is the surgical robotics challenge, (https://github.com/surgical-robotics-ai/surgical_robotics_challenge).

Comment (AkhilDeo) [reply ambf_ros_modules/examples/dvrk_control_socket_example/README.md:1]: > Hello Akhil, thanks for this PR. I think this one is more relevant to the SRC. What do you think?

Makes sense to me, moved this PR over to SRC.

**Pull request closed without merging**